spec/uploader_spec.rb

```diff
@@ -49,6 +49,16 @@
     end
   end
 
+  it "should support the cinemagraph_analysis parameter for upload and explicit" do
+    result = Cloudinary::Uploader.upload(Pathname.new(TEST_IMG), :cinemagraph_analysis => true, :tags => [TEST_TAG, TIMESTAMP_TAG])
```

Comment:
It would be better to use mock and check that `cinemagraph_analysis` is present in the request